aiohttp/client_reqrep.py

```diff
@@ -97,41 +97,45 @@ def __init__(self, method, url, *,
 
     def update_host(self, url):
         """Update destination host, port and connection type (ssl)."""
-        scheme, netloc, path, query, fragment = urllib.parse.urlsplit(url)
+        scheme, netloc, path, query, fragment = \
+            url_parsed = urllib.parse.urlsplit(url)
```

Comment:
Or even drop second line from my previous comment and use `url_parsed` everywhere in the method.